src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter
+            .ofPattern("dd.MM.yyyy");
+
+    public static String getSalaryInfo(String[] names, String[] data,
+                                       String dateFrom, String dateTo) {
+        LocalDate fromDate = null;
+        LocalDate toDate = null;
+        try {
+            fromDate = LocalDate.parse(dateFrom, DATE_FORMATTER);
+            toDate = LocalDate.parse(dateTo, DATE_FORMATTER);
+        } catch (Exception e) {
+            throw new RuntimeException("Invalid date format");
+        }
```

Comment:
LocalDate.parse method throws exception that already extends RuntimeException.
There is no need to catch such exception here and throw another one. Program will be interrupted anyway.
Please remove try/catch blocks used